tools/symflower_test.go

```diff
@@ -0,0 +1,46 @@
+package tools
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/symflower/eval-dev-quality/log"
+)
+
+func TestSymflowerInstall(t *testing.T) {
+	temporaryPath := t.TempDir()
+
+	chmodPath, err := exec.LookPath("chmod")
+	require.NoError(t, err)
+	t.Setenv("PATH", strings.Join([]string{temporaryPath, filepath.Dir(chmodPath)}, string(os.PathListSeparator)))
+
+	t.Run("Tools are not yet installed", func(t *testing.T) {
+		symflowerPath, err := exec.LookPath("symflower")
+		require.Error(t, err)
+		require.Empty(t, symflowerPath)
+	})
+
+	t.Run("Install tools for first time which should install all tools", func(t *testing.T) {
+		logOutput, log := log.Buffer()
+		require.NoError(t, SymflowerInstall(log, temporaryPath))
+
+		require.Contains(t, logOutput.String(), `Install "symflower" to`)
+		symflowerPath, err := exec.LookPath("symflower")
+		require.NoError(t, err)
+		require.NotEmpty(t, symflowerPath)
+	})
+
+	t.Run("Install tools a second time which should install no new tools", func(t *testing.T) {
+		logOutput, log := log.Buffer()
+		require.NoError(t, SymflowerInstall(log, temporaryPath))
+
+		require.NotContains(t, logOutput.String(), `Install "symflower" to`)
+		symflowerPath, err := exec.LookPath("symflower")
+		require.NoError(t, err)
+		require.NotEmpty(t, symflowerPath)
+	})
```

Comment:
Would only use `require` for the errors that are not our fault (i.e. `exec.LookPath`). And then use `assert` for the other assertions.

Also for the integration tests of the command.

Reply:
Don't agree. Require is to stop a test and assert allows to execute further.  Use require if further execution does not make sense.